lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -578,7 +580,7 @@ export abstract class ClientApplication {
     /**
      * Use to log out the current user, and redirect the user to the postLogoutRedirectUri.
      * Default behaviour is to redirect the user to `window.location.href`.
-     * @param {@link (EndSessionRequest:type)}
+     * @param logoutRequest
```

Comment:
One more inconsistency: we either add @returns for all public APIs or just don't!

Reply:
I can look at having this enforced in the js doc linting pr.